src/fields/Fields.cpp

```diff
@@ -678,7 +678,7 @@ Fields::SetBoundaryCondition (amrex::Vector<amrex::Geometry> const& geom, const
             amrex::get<0>(coeff_tuple) = 0._rt;
         }
 
-        SetDirichletBoundaries(arr_staging_area, staging_box, geom[lev],
+        SetDirichletBoundaries(arr_staging_area, staging_box, geom[lev], 1, 1,
```

Comment:
Could you add a short comment (here or below) explaining why offset and factor should be 1 here?

Reply:
The DST-Type-1-based poisson solver needs the boundary condition 1 cell next to the last cell scaled by 1/dx^2. However the MG solvers need it half a cell next to the last cell scaled by 8/(3*dx^2) (thanks to Weiqun for showing me this). The open boundary condition only works with the DST solver so 1, 1 is sufficient here.